### PR TITLE
feat(split-button): icon slot (VIV-1442)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/split-button/README.md [hunk 1]
```diff
@@ -180,6 +180,19 @@ Add the `disabled` attribute to disable the split button.
...
+### Icon
+
+Set the `icon` slot to show an icon.  
+If set, the `icon` attribute is ignored.  
+Note: Icon, by its own, doesn't make a discernible text. An `aria-label` must be provided to ensure that the user can understand the split button's purpose.
+
+
+```html preview
+<vwc-split-button appearance='outlined' label='submit'>
+		<vwc-icon slot="icon" name="check-circle-solid" connotation="success"></vwc-icon>
```
Comment (TaylorJ76) [inline libs/components/src/lib/split-button/README.md:192]: Should we add an `aria-label` here, like we advise in the text above?

**Review by TaylorJ76 — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/split-button/README.md (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/split-button/README.md:192]: yes, you are right! I'll check if we have it in the icon example as well

Comment (rachelbt) [reply libs/components/src/lib/split-button/README.md:192]: maybe the text needs to be more accurate - if only an icon is set in the button...

Comment (rachelbt) [reply libs/components/src/lib/split-button/README.md:192]: changed the text :)

**Review by TaylorJ76 — APPROVED** [review]

**Pull request merged**